example_configs/text2text/en-de/transformer-big.py

```diff
@@ -18,7 +18,7 @@
 d_model = 512
 num_layers = 6
 
-data_root = "[REPLACE THIS TO THE PATH WITH YOUR WMT DATA]"
+data_root = "./wmt16_en_dt/"
```

Comment:
Just one last thing: can you add a comment here to indicate that users should replace this path with path to their data? Or change it back to the placeholder with that message, but I would prefer having it as a comment